.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+fixes EASY-
+
+#### When applied it will
+* 
+* 
+* 
+
+#### Where should the reviewer @DANS-KNAW/dans-wicket-leaflet-lib start?
```

Comment:
Is this @dans-knaw/dans-wicket-leaflet-lib a real thing? I don't think so. Should be @DANS-KNAW/easy if you ask me...